### PR TITLE
RHIDP-3974 managing authorization using the Web UI

### PR DESCRIPTION
**Pull request by themr0c**
Some xref statements depend on https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/631.

**IMPORTANT: Do Not Merge - To be merged by Docs Team Only**

**Version(s):** 1.3.1

**Issue:** https://issues.redhat.com/browse/RHIDP-3974

**Link to docs preview:** https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-633/authorization/#proc-rbac-ui-manage-roles_title-authorization

**Reviews:**
- [x] SME: @dzemanov 
- [x] QE: @albarbaro 
- [x] Docs review: @jmagak 
- [ ] Additional review: @mention assignee (by writer)

**Additional information:**

### CODE REVIEW

**Review by albarbaro — APPROVED** [review]
lgtm

File: assemblies/assembly-managing-authorizations-by-using-the-rhdh-web-ui.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[id='proc-rbac-ui-manage-roles_{context}']
+= Managing role-based access controls (RBAC) using the {product} Web UI
+
+Policy administrators can use the {product-short} web interface (Web UI) to allocate specific roles and permissions to individual users or groups. Allocating roles ensures that access to resources and functionalities is regulated across the {product-short}.
+
+With the policy administrator role in {product-short}, you can assign permissions to users and groups, which allow users or groups to view, create, modify, and delete the roles using the {product-short} Web UI.
```
Comment (dzemanov) [inline assemblies/assembly-managing-authorizations-by-using-the-rhdh-web-ui.adoc:6]: Looks great, only have 1 suggestion for you:

```suggestion
With the policy administrator role in {product-short}, you can assign permissions to users and groups. This role allows you to view, create, modify, and delete the roles using  {product-short} Web UI.
```

**Comment by jmagak** [comment]
Hi @themr0c , LTGM.

**Review by Gerry-Forde — APPROVED** [review]

**Pull request merged**